pages/contribute/how_to_contribute.md

```diff
@@ -1,39 +1,30 @@
-* **If you are happier using Google Docs**: follow [the Google Doc way](google_doc_way).
-* **If you are familiar with Git**: fork the repo and create a pull request (see our [instructions](working_with_git)).
-* **If you just want to make a quick suggestion**: submit your comments/suggestions using the form below.
+* Follow our [style guide](https://rdmkit.elixir-europe.org/style_guide).
+* Ensure your content respects copyright. Please follow our[ Copyright guidelines](https://rdmkit.elixir-europe.org/copyright).
+* Provide acknowledgement for all your contributions.
 
-## Contributor responsibilities
 
-When writing for this website keep in mind the following:
+## Ways of contributing
 
-* For the sake of consistency, please follow our [style guide](style_guide).
-* It is your responsibility to make sure that the content you provide respects copyright. Please follow our [Copyright guidelines](copyright).
-* Please read our [Markdown cheat sheet](markdown_cheat_sheet) in order to be able to fill in the page metadata and to format the layout correctly.
-* To avoid having to resolve many issues during the revision process, please check the [Editors checklist](editors_checklist) and make sure you have complied with the requirements.
-* If others were involved in your contribution, by writing up or by providing resources such as diagrams or links, please make sure you acknowledge them in the contributors' section of your page, after receiving their permission.
+* Google Doc. If you are unfamiliar with Git and prefer to use a classic document editor, follow [the Google Doc way](https://rdmkit.elixir-europe.org/google_doc_way)
+* GitHub. If you are unfamiliar with Git but want to try contributing on GitHub with no technical knowledge required, follow [our step-by-step instructions](https://rdmkit.elixir-europe.org/github_way)
+* Git. If you are familiar with Git, fork the repo and create a pull request following our [instructions](https://rdmkit.elixir-europe.org/working_with_git)
 
-## Acknowledgement of contributors
 
-Contributors will be shown at the bottom of the page if listed in the metadata of the markdown file. All contributors will also be displayed in the [contributors page](contributors). If you want to link your GitHub account, ORCID or email address, please add your name and corresponding information to the [CONTRIBUTORS file](https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml).
+## Acknowledgement and ownership of content
 
-## Ownership of content
+Contributors will be shown at the bottom of each page and on the main [contributors page](https://rdmkit.elixir-europe.org/contributors). if listed in the metadata of the markdown file. All contributors will also be displayed on the. If you want to link your GitHub account, ORCID or email address, please add your name and corresponding information to the [CONTRIBUTORS file](https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml).
```

Comment:
> All contributors will also be displayed on the.

This sentence does not seem to be complete

Reply:
I'm guessing that's a fragment that shouldn't be there. Maybe the passage should read:

> Contributors will be shown at the bottom of each page and on the main [contributors page](https://rdmkit.elixir-europe.org/contributors) if listed in the metadata of the markdown file. If you want to link your GitHub account, ORCID or email address, please add your name and corresponding information to the [CONTRIBUTORS file](https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml).

Reply:
> I'm guessing that's a fragment that shouldn't be there.

I think you are right.

> If you want to link your GitHub account, ORCID or email address, please add your name and corresponding information to the [CONTRIBUTORS file](https://github.com/elixir-europe/rdmkit/blob/master/_data/CONTRIBUTORS.yaml).

I never realised adding your information to the CONTRIBUTORS yaml was not mandatory. I think we should require some information (at least the ORCID) since we are promoting best practices. Any thoughts @bedroesb?

Reply:
Can we assume everyone has a ORCID?

Reply:
> Can we assume everyone has an ORCID?

It would be my assumption, but maybe I am living inside a bubble. ORCID is typically required by scientific journals to publish, so at least there is a strong push for scientists to create one anyway. (I created mine 7 years ago)